### PR TITLE
Add Documentation for Snapshot Feature

### PR DESCRIPTION
**Pull request by xing-yang**
kubernetes/features#177

### CODE REVIEW

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -625,6 +625,34 @@ Volume binding matrix for statically provisioned volumes:
...
 {{< /note >}}
 
+## Volume Snapshot and Restore Volume from Snapshot Support
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Volume snapshot feature was added to support CSI Volume Plugins only. For details, see [volume snapshots](/docs/concepts/storage/volume-snapshots/).
+
+To enable support for restoring a volume from a volume snapshot data source, enable the
+`VolumeSnapshotDataSource` feature gate on the apiserver, controller-manager and the kubelet.
```
Comment (jingxu97) [inline content/en/docs/concepts/storage/persistent-volumes.md]: I think kubelet does not need the feature gate?

Comment (xing-yang) [reply content/en/docs/concepts/storage/persistent-volumes.md]: I think feature gate is needed by kubelet for some other features, but we can remove it from this one.

**Comment by zparnold** [comment]
@xing-yang Made a few edits for grammar. PTAL and let me know if it works for you. Otherwise:
/approve

**Comment by xing-yang** [comment]
@zparnold Your edits look good.  Thanks!

**Comment by zparnold** [comment]
Cool! Thanks!
/lgtm

**Pull request merged**